### PR TITLE
Translation from "Quarkus security vulnerability detection"

### PR DESCRIPTION
**Pull request by alvescamila87**
Hey guys! :)
Following my translation related to the Guide about "Security vulnerability detection".

### CODE REVIEW

File: l10n/po/pt_BR/_guides/security-overview.adoc.po [hunk 1]
```diff
@@ -3,264 +3,260 @@
...
 msgid "Key features of Quarkus Security"
 msgstr "Principais recursos do Quarkus Security"
 
 #: _guides/security-overview.adoc
-#, fuzzy
 msgid ""
 "The Quarkus Security framework provides built-in security authentication mechanisms for Basic, Form-based, and mutual TLS (mTLS) authentication.\n"
 "You can also use other well-known xref:security-authentication-mechanisms.adoc#other-supported-authentication-mechanisms[authentication mechanisms], such as OpenID Connect (OIDC) and WebAuthn."
-msgstr "A estrutura do Quarkus Security fornece mecanismos de autenticação de segurança incorporados para autenticação básica, baseada em formulário e TLS mútua (mTLS). O senhor também pode usar outros xref:security-authentication-mechanisms.adoc#other-supported-authentication-mechanisms[mecanismos de autenticação] conhecidos, como o OpenID Connect (OIDC) e o WebAuthn."
+msgstr ""
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/security-overview.adoc.po:44]: Can you remove this line break?

File: l10n/po/pt_BR/_guides/security-overview.adoc.po [hunk 2]
```diff
@@ -3,264 +3,260 @@
...
+msgstr "Os mecanismos de autenticação dependem dos xref:security-identity-providers.adoc[Identity providers] para verificar as credenciais de autenticação e mapeá-las para uma instância `SecurityIdentity` com o nome de usuário, os papéis, as credenciais de autenticação originais e outros atributos."
 
 #: _guides/security-overview.adoc
-#, fuzzy
 msgid ""
 "{project-name} also includes built-in security to allow for role-based access control (RBAC) based on the common security annotations `@RolesAllowed`, `@DenyAll`, `@PermitAll` on REST endpoints, and Contexts and Dependency Injection (CDI) beans.\n"
 "For more information, see the Quarkus xref:security-authorize-web-endpoints-reference.adoc[Authorization of web endpoints] guide."
-msgstr "{project-name} também inclui segurança integrada para permitir o controle de acesso baseado em função (RBAC) com base nas anotações de segurança comuns `@RolesAllowed` , `@DenyAll` , `@PermitAll` em endpoints REST e beans de injeção de dependência e contextos (CDI). Para obter mais informações, consulte o guia Quarkus xref:security-authorize-web-endpoints-reference.adoc[Authorization of web endpoints] ."
+msgstr ""
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/security-overview.adoc.po:56]: Same here

File: l10n/po/pt_BR/_guides/security-overview.adoc.po [hunk 3]
```diff
@@ -3,264 +3,260 @@
...
-msgstr "xref:security-vulnerability-detection.adoc[Detecção de vulnerabilidades de segurança e registro no National Vulnerability Database (NVD)]"
+msgstr "xref:security-vulnerability-detection.adoc[Security vulnerability detection e registro US National Vulnerability Database (NVD)]"
 
 #: _guides/security-overview.adoc
-#, fuzzy
 msgid ""
 "Quarkus Security is also highly customizable.\n"
 "For more information, see the Quarkus xref:security-customization.adoc[Security tips and tricks] guide."
-msgstr "O Quarkus Security também é altamente personalizável. Para obter mais informações, consulte o guia de xref:security-customization.adoc[dicas e truques] do Quarkus xref:security-customization.adoc[Security] ."
+msgstr ""
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/security-overview.adoc.po:115]: Same here

File: l10n/po/pt_BR/_guides/security-overview.adoc.po [hunk 4]
```diff
@@ -3,264 +3,260 @@
...
-msgstr "Compartilhamento de recursos entre origens"
+msgstr "Cross-Origin Resource Sharing (CORS)"
 
 #: _guides/security-overview.adoc
-#, fuzzy
 msgid ""
 "To make your Quarkus application accessible to another application running on a different domain, you need to configure cross-origin resource sharing (CORS).\n"
 "For more information about the CORS filter Quarkus provides, see the xref:security-cors.adoc#cors-filter[CORS filter] section of the Quarkus \"Cross-origin resource sharing\" guide."
-msgstr "Para tornar seu aplicativo Quarkus acessível a outro aplicativo em execução em um domínio diferente, é necessário configurar o compartilhamento de recursos entre origens (CORS). Para obter mais informações sobre o filtro CORS fornecido pelo Quarkus, consulte a seção de xref:security-cors.adoc#cors-filter[filtro CORS] do guia \"Compartilhamento de recursos entre origens\" do Quarkus."
+msgstr ""
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/security-overview.adoc.po:163]: Same here

File: l10n/po/pt_BR/_guides/security-overview.adoc.po [hunk 5]
```diff
@@ -3,264 +3,260 @@
...
-msgstr "Prevenção de falsificação de solicitações entre sites (CSRF)"
+msgstr "Prevenção de Cross-Site Request Forgery (CSRF)"
 
 #: _guides/security-overview.adoc
-#, fuzzy
 msgid ""
 "Quarkus Security provides a RESTEasy Reactive filter that can protect your applications against a https://owasp.org/www-community/attacks/csrf[Cross-Site Request Forgery] attack.\n"
 "For more information, see the Quarkus xref:security-csrf-prevention.adoc[Cross-Site Request Forgery Prevention] guide."
-msgstr "O Quarkus Security fornece um filtro RESTEasy Reactive que pode proteger seus aplicativos contra um ataque link:https://owasp.org/www-community/attacks/csrf[de falsificação de solicitação entre sites] . Para obter mais informações, consulte o guia Quarkus xref:security-csrf-prevention.adoc[Cross-Site Request Forgery Prevention] ."
+msgstr ""
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/security-overview.adoc.po:177]: And here

File: l10n/po/pt_BR/_guides/security-overview.adoc.po [hunk 6]
```diff
@@ -3,264 +3,255 @@
...
-msgstr "Compartilhamento de recursos entre origens"
+msgstr "Cross-Origin Resource Sharing (CORS)"
 
 #: _guides/security-overview.adoc
-#, fuzzy
 msgid ""
 "To make your Quarkus application accessible to another application running on a different domain, you need to configure cross-origin resource sharing (CORS).\n"
 "For more information about the CORS filter Quarkus provides, see the xref:security-cors.adoc#cors-filter[CORS filter] section of the Quarkus \"Cross-origin resource sharing\" guide."
-msgstr "Para tornar seu aplicativo Quarkus acessível a outro aplicativo em execução em um domínio diferente, é necessário configurar o compartilhamento de recursos entre origens (CORS). Para obter mais informações sobre o filtro CORS fornecido pelo Quarkus, consulte a seção de xref:security-cors.adoc#cors-filter[filtro CORS] do guia \"Compartilhamento de recursos entre origens\" do Quarkus."
+"Para tornar sua aplicação Quarkus acessível a partir de outra aplicação em execução em um domínio diferente, é necessário configurar o Cross-Origin Resource Sharing (CORS). \n"
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/security-overview.adoc.po:160]:
```suggestion
msgstr "Para tornar sua aplicação Quarkus acessível a partir de outra aplicação em execução em um domínio diferente, é necessário configurar o Cross-Origin Resource Sharing (CORS). \n"
```

File: l10n/po/pt_BR/_guides/security-overview.adoc.po [hunk 7]
```diff
@@ -3,264 +3,255 @@
...
-msgstr "Prevenção de falsificação de solicitações entre sites (CSRF)"
+msgstr "Prevenção de Cross-Site Request Forgery (CSRF)"
 
 #: _guides/security-overview.adoc
-#, fuzzy
 msgid ""
 "Quarkus Security provides a RESTEasy Reactive filter that can protect your applications against a https://owasp.org/www-community/attacks/csrf[Cross-Site Request Forgery] attack.\n"
 "For more information, see the Quarkus xref:security-csrf-prevention.adoc[Cross-Site Request Forgery Prevention] guide."
-msgstr "O Quarkus Security fornece um filtro RESTEasy Reactive que pode proteger seus aplicativos contra um ataque link:https://owasp.org/www-community/attacks/csrf[de falsificação de solicitação entre sites] . Para obter mais informações, consulte o guia Quarkus xref:security-csrf-prevention.adoc[Cross-Site Request Forgery Prevention] ."
+"O Quarkus Security fornece um filtro RESTEasy Reactive que pode proteger seus aplicativos contra um ataque link:https://owasp.org/www-community/attacks/csrf[Cross-Site Request Forgery].\n"
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/security-overview.adoc.po:173]:
```suggestion
msgstr "O Quarkus Security fornece um filtro RESTEasy Reactive que pode proteger seus aplicativos contra um ataque link:https://owasp.org/www-community/attacks/csrf[Cross-Site Request Forgery].\n"
```

File: l10n/po/pt_BR/_guides/security-overview.adoc.po [hunk 8]
```diff
@@ -3,264 +3,255 @@
...
 msgstr "Segredos nas propriedades do ambiente"
 
 #: _guides/security-overview.adoc
-#, fuzzy
 msgid ""
 "Quarkus provides support to store secrets in environment properties.\n"
 "For more information, see the Quarkus xref:config.adoc#secrets-in-environment-properties[store secrets in an environment properties file] guide."
-msgstr "O Quarkus oferece suporte para armazenar segredos nas propriedades do ambiente. Para obter mais informações, consulte o guia Quarkus xref:config.adoc#secrets-in-environment-properties[store secrets in an environment properties file] ."
+msgstr ""
+"O Quarkus oferece suporte para armazenar segredos nas propriedades do ambiente. \n"
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/security-overview.adoc.po:211]:
```suggestion
msgstr "O Quarkus oferece suporte para armazenar segredos nas propriedades do ambiente. \n"
```

File: l10n/po/pt_BR/_guides/security-vulnerability-detection.adoc.po [hunk 9]
```diff
@@ -66,52 +61,46 @@ msgstr "Detectar vulnerabilidades no Quarkus no momento da compilação"
...
 msgstr "Para detectar problemas menos graves, ajuste o valor de `failBuildOnCVSS` para suprimir os falsos positivos, conforme demonstrado no exemplo de código a seguir:"
 
 #: _guides/security-vulnerability-detection.adoc
-#, fuzzy
 msgid ""
 "Ensure that you review and update the suppression list regularly to ensure that the results are up to date.\n"
 "You can optionally apply a time limit to individual suppressions by adding an expiry attribute, as outlined in the following example:"
-msgstr "Certifique-se de revisar e atualizar a lista de supressão regularmente para garantir que os resultados estejam atualizados. Opcionalmente, o senhor pode aplicar um limite de tempo a supressões individuais adicionando um atributo de expiração, conforme descrito no exemplo a seguir:"
+msgstr ""
+"Certifique-se de revisar e atualizar a lista de supressão regularmente para garantir que os resultados estejam atualizados. \n"
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/security-vulnerability-detection.adoc.po:92]:
```suggestion
msgstr "Certifique-se de revisar e atualizar a lista de supressão regularmente para garantir que os resultados estejam atualizados. \n"
```

**Comment by dellamas** [comment]
Galtaldi, what is left to conclude this translation? I'm available to help.

File: l10n/po/pt_BR/_guides/security-overview.adoc.po [hunk 10]
```diff
@@ -3,264 +3,251 @@
...
 msgid "Key features of Quarkus Security"
 msgstr "Principais recursos do Quarkus Security"
 
 #: _guides/security-overview.adoc
-#, fuzzy
 msgid ""
 "The Quarkus Security framework provides built-in security authentication mechanisms for Basic, Form-based, and mutual TLS (mTLS) authentication.\n"
 "You can also use other well-known xref:security-authentication-mechanisms.adoc#other-supported-authentication-mechanisms[authentication mechanisms], such as OpenID Connect (OIDC) and WebAuthn."
-msgstr "A estrutura do Quarkus Security fornece mecanismos de autenticação de segurança incorporados para autenticação básica, baseada em formulário e TLS mútua (mTLS). O senhor também pode usar outros xref:security-authentication-mechanisms.adoc#other-supported-authentication-mechanisms[mecanismos de autenticação] conhecidos, como o OpenID Connect (OIDC) e o WebAuthn."
+"O framework Quarkus Security fornece mecanismos de autenticação de segurança nativos para autenticação Básica, baseada em Formulário e TLS mútua (mTLS).\n"
```
Comment (gastaldi) [inline l10n/po/pt_BR/_guides/security-overview.adoc.po:44]:
```suggestion
msgstr "O framework Quarkus Security fornece mecanismos de autenticação de segurança nativos para autenticação Básica, baseada em Formulário e TLS mútua (mTLS).\n"
```

Comment (gastaldi) [reply l10n/po/pt_BR/_guides/security-overview.adoc.po:44]: Please make sure that `msgstr` token is not removed from the other parts

**Review by gastaldi — APPROVED** [review]

**Comment by gastaldi** [comment]
Looks good now. Can you resolve the conflicts before this gets merged? Thanks!

**Pull request merged**